_episodes/08-putting-it-all-together.md

```diff
@@ -86,32 +89,37 @@ also change the name of the columns in the DataFrame like this:
 ~~~
 {: .language-python}
 
-## Make a line plot of your data
+## Matplotlib plot library
 
-Matplotlib is a Python library that can be used to visualize data. The
-toolbox `matplotlib.pyplot` is a collection of functions that make matplotlib
-work like MATLAB. In most cases, this is all that you will need to use, but
-there are many other useful tools in matplotlib that you should explore.
+[Matplotlib](http://matplotlib.org/) is a Python package used widely throughout the scientific Python community to produce high quality and publication-ready graphics. It supports a wide range of output formats including PNG (and other raster formats), PostScript/EPS, PDF and SVG. 
```

Comment:
Title says `Matplotlib plot *library*`, the first sentence says ...`Python *package*`. It *might* be useful to use one or the other but not both.

Reply:
`high-quality` because it refers to `graphics`